distribution/src/bin-regular/hz-start.bat

```diff
@@ -1,51 +1,58 @@
-    echo JAVA could not be found in your system.
-    echo Please install Java 1.8 or higher!!!
-    exit /b 2
-)
-
-REM ******* you can enable following variables by uncommenting them
-
-REM ******* minimum heap size
-REM set MIN_HEAP_SIZE=1G
-
-REM ******* maximum heap size
-REM set MAX_HEAP_SIZE=1G
-
-
-if NOT "%MIN_HEAP_SIZE%" == "" (
-	set JAVA_OPTS=%JAVA_OPTS% -Xms%MIN_HEAP_SIZE%
-)
-
-if NOT "%MAX_HEAP_SIZE%" == "" (
-	set JAVA_OPTS=%JAVA_OPTS% -Xmx%MAX_HEAP_SIZE%
-)
-
-FOR /F "tokens=* USEBACKQ" %%F IN (`CALL "%RUN_JAVA%" -cp "%~dp0..\lib\hazelcast-all-${project.version}.jar" com.hazelcast.internal.util.JavaVersion`) DO SET JAVA_VERSION=%%F
-
-IF NOT "%JAVA_VERSION%" == "8" (
-	set JAVA_OPTS=%JAVA_OPTS% --add-modules java.se --add-exports java.base/jdk.internal.ref=ALL-UNNAMED --add-opens java.base/java.lang=ALL-UNNAMED --add-opens java.base/java.nio=ALL-UNNAMED --add-opens java.base/sun.nio.ch=ALL-UNNAMED --add-opens java.management/sun.management=ALL-UNNAMED --add-opens jdk.management/com.sun.management.internal=ALL-UNNAMED
-)
-
-set "CLASSPATH=%~dp0..\lib\hazelcast-all-${project.version}.jar;%~dp0..\bin\user-lib;%~dp0..\bin\user-lib\*"
-
-ECHO ########################################
-ECHO # RUN_JAVA=%RUN_JAVA%
-ECHO # JAVA_OPTS=%JAVA_OPTS%
-ECHO # starting now...."
-ECHO ########################################
-
-"%RUN_JAVA%" %JAVA_OPTS% -cp "%CLASSPATH%" "com.hazelcast.core.server.HazelcastMemberStarter"
-
-ENDLOCAL
+@echo off
+
+SETLOCAL
+
+if "x%JAVA_HOME%" == "x" (
+    echo JAVA_HOME environment variable not available.
+    set RUN_JAVA=java
+) else (
+    set "RUN_JAVA=%JAVA_HOME%\bin\java"
+)
+
+"%RUN_JAVA%" -version 1>nul 2>nul || (
+    echo JAVA could not be found in your system.
+    echo Please install Java 1.8 or higher!!!
+    exit /b 2
+)
+set HAZELCAST_HOME=%~dp0..
+
+REM ******* you can enable following variables by uncommenting them
+
+REM ******* minimum heap size
+REM set MIN_HEAP_SIZE=1G
+
+REM ******* maximum heap size
+REM set MAX_HEAP_SIZE=1G
+
+
+if NOT "%MIN_HEAP_SIZE%" == "" (
+	set JAVA_OPTS=%JAVA_OPTS% -Xms%MIN_HEAP_SIZE%
+)
+
+if NOT "%MAX_HEAP_SIZE%" == "" (
+	set JAVA_OPTS=%JAVA_OPTS% -Xmx%MAX_HEAP_SIZE%
+)
+
+FOR /F "tokens=* USEBACKQ" %%F IN (`CALL "%RUN_JAVA%" -cp "%HAZELCAST_HOME%\lib\hazelcast-${project.version}.jar" com.hazelcast.internal.util.JavaVersion`) DO SET JAVA_VERSION=%%F
+
+IF NOT "%JAVA_VERSION%" == "8" (
+	set JAVA_OPTS=%JAVA_OPTS% --add-modules java.se --add-exports java.base/jdk.internal.ref=ALL-UNNAMED --add-opens java.base/java.lang=ALL-UNNAMED --add-opens java.base/java.nio=ALL-UNNAMED --add-opens java.base/sun.nio.ch=ALL-UNNAMED --add-opens java.management/sun.management=ALL-UNNAMED --add-opens jdk.management/com.sun.management.internal=ALL-UNNAMED
+)
+
+set JAVA_OPTS=%JAVA_OPTS%^
+ "-Dhazelcast.logging.type=log4j2"^
+ "-Dlog4j.configurationFile=file:%HAZELCAST_HOME%\config\log4j2.properties"^
+ "-Dhazelcast.config=%HAZELCAST_HOME%\config\hazelcast.yaml"^
```

Comment:
I see these are new additions, but I don't see this change explicitly mentioned in the description or in the issues this PR fixes. I recall we had a discussion on making YAML the primary config language, but I don't know the outcome. Just want to make sure this is in line with that discussion. Do you know this @mmedenjak?

Reply:
I guess these were inherited from [`jet-start.bat`](https://github.com/hazelcast/hazelcast-jet/blob/2324c0a1e688a4511826e0f5cacbe12a886b9f89/hazelcast-jet-distribution/src/bin-regular/jet-start.bat#L9-L14).

I'll remove the config property, leave the rest.

Reply:
Removed from both start scripts.